src/subgraph/accounts/subgraphQueries.ts

```diff
@@ -11,3 +11,67 @@ export const fetchRealizedPnlData = (userAddress: string) => gql`
     }
   }
 `;
+
+/// Fetch Portfolio total for a set of addresses
+/// Portfolio total consists of
+/// 1. Realized PNL from positions and vaults - user.totalRealizedPnlPositions, user.totalRealizedPnlVaults
+/// 2. Unrealized PNL from positions - positions.netUnrealizedPnlInUsd
+/// 3. Deposited liquidity in vaults -
+/// 4. Deposited collateral of all open positions - positions.positionCollateral
+export const fetchPortfolioData = (userAddresses: string[]) => gql`
+{
+  accounts(where: {id_in: [${userAddresses.map((id) => `"${id}"`).join(', ')}]}) {
+    id
+    totalRealizedPnlPositions
+    totalRealizedPnlVaults
+  }
+  positions(
+    first: 1000
```

Comment:
Subgraphs would return a maximum of 1000 records per query. If we need more than that, we have to add pagination logic.
Here, as we are fetching only OPEN positions, there's a very low chance that a user will have more than 1000 open positions. If we encounter such issues, we can add pagination later.

Reply:
Edit: This will return max. 1000 records for all the users, so we have to keep the number of user addresses lower to prevent this issue. I will add the pagination logic later in a separate PR